addons/mrp/models/stock_rule.py

```diff
@@ -106,6 +106,24 @@ def _get_date_planned(self, product_id, company_id, values):
         date_planned = date_planned - relativedelta(days=company_id.manufacturing_lead)
         return date_planned
 
+    def _get_lead_days(self, product):
+        """Extend the fonction in order to add the product and company
+        manufacture delay to the cumulative delay and its description.
+        """
+        delay, delay_description = super()._get_lead_days(product)
+        manufature_rule = self.filtered(lambda r: r.action == 'manufacture')
```

Comment:
i know it should not happen but if you create two manufacture rules, this will fail